p/sbb/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const products = require('./products')
+
+const transformReqBody = (ctx, body) => {
+	// sourced from https://github.com/marudor/BahnhofsAbfahrten/blob/1c99d8b/packages/server/HAFAS/profiles.ts#L344-L361
+	body.client = {
+		type: 'AND',
+		id: 'DBZUGRADARNETZ',
+		v: '',
+	}
+	body.ver = '1.18'
+	body.auth = { aid: 'hf7mcf9bv3nv8g5f', type:'AID' }
+	body.lang = 'deu'
+
+	return body
+}
+
+const sbbProfile = {
+	locale: 'de-CH',
+	timezone: 'Europe/Zurich',
+	endpoint: 'http://fahrplan.sbb.ch/bin/mgate.exe',
```

Comment:
The `https:` endpoint actually redirects all requests to `http:`. 😢

Reply:
:( I guess we should hardcode `https://` then, in the hopes that this is changed at some point

Reply:
Please try if the `fetch` API then re-sends the body for the 2nd request; If it doesn't, there's probably a good reason not to send the body after a redirect.